tests/test_global_opts.rs

```diff
@@ -235,5 +235,30 @@ fn test_help() {
           --at-operation <AT_OPERATION>  Operation to load the repo at [default: @] [aliases: at-op]
           --color <WHEN>                 When to colorize output (always, never, auto)
           --config-toml <TOML>           Additional configuration options
+      -v, --verbose                      Enable verbose logging
     "###);
 }
+
+#[test]
+fn test_verbose_logging_enabled() {
```

Comment:
I don't see this test running in the CI output. Am I missing something, or is CI not hitting these for some reason? I ask because I'm not sure if the tracing `fmt` layer will include ANSI on windows by default, which might make this test fragile.

Reply:
I can find them in the build runs for Windows and Linux (line 600 and line 567 of latest actions respectively). Make sure to use the actions search bar and not browser search.